tests/containers/container_engine.pm

```diff
@@ -147,7 +147,7 @@ sub basic_container_tests {
         # Default OCI runtime should be runc on all products, SUSE & openSUSE
         my $template = ($runtime eq "podman") ? "{{ .Host.OCIRuntime.Name }}" : "{{ .DefaultRuntime }}";
         my $runtime = script_output("$runtime info -f '$template'");
-        my $expected = is_sle_micro('>=6.0') ? "crun" : "runc";
+        my $expected = is_sle_micro('>=6.1') ? "crun" : "runc";
```

Comment:
LGTM
Also it could have been 
```suggestion
        my $expected = is_sle_micro('>6.0') ? "crun" : "runc";
```
🙂

Reply:
This is wrong, SLEM 6.2 has runc again. Only SLEM 6.1 will have crun.